doc/user_guide/interactions.rst

```diff
@@ -695,12 +719,125 @@ often provides a more convenient syntax
 for simple interactions like this one
 since they can also be accessed in expression strings
 as we saw above.
-Selections and parameters can be used anywhere where expressions are valid, for
-example, in a :ref:`user-guide-calculate-transform` or a
-:ref:`user-guide-filter-transform` transform.
+Similarly,
+it is often possible to use equality statements
+such as ``alt.datum.xval == selector`` to lookup exact values
+but it is often more convenient to switch to a selection parameter
+and specifying a field/encoding.
+
+Logic-Driven Comparisons
+------------------------
+
+Data-driven lookups and comparisons are very helpful
+but sometimes we might want to make a logic comparison
+in the conditional statement instead
+that is independent of the data.
```

Comment:
```suggestion
A logic comparison is a type of comparison
that is based on logical rules and conditions,
rather than on the actual data values themselves.
```